### PR TITLE
Wording changes to improve clarity.

### PR DESCRIPTION
**Pull request by colinedwardwood**
**What is this feature?**

[Add a brief description of what the feature or update does.]

**Why do we need this feature?**

[Add a description of the problem the feature is trying to solve.]

**Who is this feature for?**

[Add information on what kind of user the feature is for.]

**Which issue(s) does this PR fix?**:

Fixes #

**Special notes for your reviewer:**

Please check that:
- [ ] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [ ] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/#how-to-determine-if-content-belongs-in-a-whats-new-document), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/) doc.

### CODE REVIEW

File: docs/sources/setup-grafana/set-up-for-high-availability.md [hunk 1]
```diff
@@ -13,18 +13,19 @@ weight: 900
 
 # Set up Grafana for high availability
 
-Setting up Grafana for high availability is fairly simple. You just need a shared database for storing dashboard, users,
-and other persistent data. So the default embedded SQLite database will not work, you will have to switch to MySQL or Postgres.
+By default, Grafana uses an embedded sqlite3 database to store users, dashboards, and other persistent data. To set Grafana up for high availability a shared database is required to store this data. This shared database can be either MySQL or Postgres.
```
Comment (chri2547) [inline docs/sources/setup-grafana/set-up-for-high-availability.md:16]:
```suggestion
Grafana uses an embedded sqlite3 database to store users, dashboards, and other persistent data by default. For high availability, you must use a shared database to store this data. This shared database can be either MySQL or Postgres.
```

File: docs/sources/setup-grafana/set-up-for-high-availability.md [hunk 2]
```diff
@@ -13,18 +13,19 @@ weight: 900
...
 
 <div class="text-center">
   <img src="/static/img/docs/tutorials/grafana-high-availability.png"  max-width= "800px" class="center" />
 </div>
 
-## Configure multiple servers to use the same database
+## Pre-requisite Setup
```
Comment (chri2547) [inline docs/sources/setup-grafana/set-up-for-high-availability.md:22]:
```suggestion
## Before you begin 
```

File: docs/sources/setup-grafana/set-up-for-high-availability.md [hunk 3]
```diff
@@ -13,18 +13,19 @@ weight: 900
...
   <img src="/static/img/docs/tutorials/grafana-high-availability.png"  max-width= "800px" class="center" />
 </div>
 
-## Configure multiple servers to use the same database
+## Pre-requisite Setup 
 
-First, you need to set up MySQL or Postgres on another server and configure Grafana to use that database.
-You can find the configuration for doing that in the [[database]]({{< relref "configure-grafana/#database" >}}) section in the Grafana config.
-Grafana will now persist all long term data in the database. How to configure the database for high availability is out of scope for this guide. We recommend finding an expert on the database you're using.
+In order to proceed with setting up Grafana for high availability you will need to first configure a MySQL or Postgres database. It is recommended that this database be configured to be highly available. Configuring the database itself for high availability is out of scope for this guide, but instructions can be found online for each database.
```
Comment (chri2547) [inline docs/sources/setup-grafana/set-up-for-high-availability.md:24]:
```suggestion
```

File: docs/sources/setup-grafana/set-up-for-high-availability.md [hunk 4]
```diff
@@ -13,18 +13,19 @@ weight: 900
...
+
+## Configure multiple Grafana servers to use the same database
+
+Once you have either a Postgres or MySQL database available, you can now configure your multiple Grafana instances to use a shared backend database. Grafana has default and custom configuration files and you can update the database settings by updating your custom configuration file as described in the [[database]]({{< relref "configure-grafana/#database" >}}) section of the Grafana configuration. Once configured to use a shared database, your multiple Grafana instances will now persist all long term data in that database.
```
Comment (chri2547) [inline docs/sources/setup-grafana/set-up-for-high-availability.md:28]:
```suggestion
Once you have a Postgres or MySQL database available, you can configure your multiple Grafana instances to use a shared backend database. Grafana has default and custom configuration files, and you can update the database settings by updating your custom configuration file as described in the [[database]]({{< relref "configure-grafana/#database" >}}). Once configured to use a shared database, your multiple Grafana instances will persist all long-term data in that database.
```

**Review by chri2547 — COMMENTED** [review]
@colinedwardwood A couple of suggestions. Pls review to ensure I didn't change the technical meaning of what you wrote. Thanks for the contribution! Ping when you're done and I'll approve and merge.

Question: I can backport this change to 9.2, 9.3, 9.4 and 9.5. I assume that the change applies to all those versions?

Also, you'll need to accept the CLA. Just click the CLAssistant button above.

**Comment by colinedwardwood** [comment]
Thanks for the corrections. You didn't change any technical meaning. I accepted the CLA, should be good to merge.

File: docs/sources/setup-grafana/set-up-for-high-availability.md [hunk 5]
```diff
@@ -13,18 +13,20 @@ weight: 900
...
 
-## Configure multiple servers to use the same database
+## Before you begin 
 
-First, you need to set up MySQL or Postgres on another server and configure Grafana to use that database.
-You can find the configuration for doing that in the [[database]]({{< relref "configure-grafana/#database" >}}) section in the Grafana config.
-Grafana will now persist all long term data in the database. How to configure the database for high availability is out of scope for this guide. We recommend finding an expert on the database you're using.
+Before you complete the following tasks, configure a MySQL or Postgres database to be highly available. Configuring the MySQL or Postgres database for high availability is out of the scope of this guide, but you can find instructions online for each database.
+
+
```
Comment (chri2547) [inline docs/sources/setup-grafana/set-up-for-high-availability.md:26]:
```suggestion
```

File: docs/sources/setup-grafana/set-up-for-high-availability.md [hunk 6]
```diff
@@ -13,18 +13,19 @@ weight: 900
...
-## Configure multiple servers to use the same database
+## Before you begin 
 
-First, you need to set up MySQL or Postgres on another server and configure Grafana to use that database.
-You can find the configuration for doing that in the [[database]]({{< relref "configure-grafana/#database" >}}) section in the Grafana config.
-Grafana will now persist all long term data in the database. How to configure the database for high availability is out of scope for this guide. We recommend finding an expert on the database you're using.
+Before you complete the following tasks, configure a MySQL or Postgres database to be highly available. Configuring the MySQL or Postgres database for high availability is out of the scope of this guide, but you can find instructions online for each database.
+
+## Configure multiple Grafana servers to use the same database
+
```
Comment (chri2547) [inline docs/sources/setup-grafana/set-up-for-high-availability.md:27]:
```suggestion
```

**Review by chri2547 — APPROVED** [review]

**Review by chri2547 — APPROVED** [review]

**Pull request merged**